_posts/2021-03-03-geoserver-2-19-rc-released.md

```diff
@@ -0,0 +1,73 @@
+- Announcements
+tags:
+- Release Candidate
+---
+
+We are pleased to announce the release of GeoServer [2.17.5](http://geoserver.org/release/2.17.5/) with downloads (
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.17.5/geoserver-2.17.5-war.zip/download) |
+[bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.17.5/geoserver-2.17.5-bin.zip/download) ), [documentation](https://sourceforge.net/projects/geoserver/files/GeoServer/2.17.5/geoserver-2.17.5-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.17.5/extensions/) .
+
+We are happy to announce GeoServer [2.19-RC](http://geoserver.org/release/2.19-RC/) release candidate is available for testing. Downloads are available ([zip](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-bin.zip/download) and [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-war.zip/download)) along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/extensions/).
+
+This is a GeoServer release candidate made in conjunction with GeoTools 25-RC and GeoWebCache 1.19-RC.
+
+  * Release candidates are a community building exercise and are not intended for production use.
+  * We ask the community (everyone: individuals, organizations, service providers) to download and thoroughly test this release candidate and report back.
+  * Participating in testing release candidates is a key expectation of our [open source social contract](http://www.ianturton.com/talks/foss4g.html#/). We make an effort to thank each person who tests in our release announcement and project presentations!
+
+## Release Candidate Testing Priorities
+
+This is an exciting release and a lot of great new functionality has been added. We would like to ask for your assistance testing the following:
+
+  * The number one testing priority is to try out GeoServer with your data! _Mass market open source thrives on having many people to review. Scientific open source like GeoServer thrives on exposure to many datasets_.
+  * Help check that new [extension download bundles](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/extensions/) have contain everything needed, including appropriate readme instructions and open source license information.
+  * The rest of this blog post highlights new features for GeoServer 2.19, please try out these features, read the documentation links, and ask questions.
+
+Known Issues:
+
+  * Layer configured with missing style throws NPE
+
+## MapML module graduated to extension
+
+...
+
+## Retired Extensions
+
+Several extensions being retired from this release:
+
+  * ArcSDE
+  
+In each case we found that the extension is no longer used, or lacked sufficient feedback and resources for continued development.
```

Comment:
"We found that the extension is no longer used, and lacked sufficient feedback and resources for continued development.